End2End/functest/src/main/java/com/microsoft/azure/functions/endtoend/ServiceBusTopic.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.azure.functions.endtoend;
+
+import com.microsoft.azure.functions.ExecutionContext;
+import com.microsoft.azure.functions.annotation.BindingName;
+import com.microsoft.azure.functions.OutputBinding;
+
+public class ServiceBusTopic {
+
+    public static void serviceBusTopicTrigger(
+        @BindingName("message") String message, 
```

Comment:
Changes looks good. We do need to update all the tests to use annotations and delete checked in function.json files.